spec/features/view_scheduled_messages_spec.rb

```diff
@@ -44,24 +44,44 @@
     expect(page).to have_content(second_scheduled_message.title)
     expect(page).to have_content(second_scheduled_message.body)
     expect(page).to have_content(second_scheduled_message.days_after_start)
-    expect(page).to have_content(second_scheduled_message.time_of_day.strftime("%l:%M %p"))
+    expect(page).to have_content(get_formatted_time(second_scheduled_message))
 
     expect(page).to have_content("Prev")
     expect(page).to have_content("First")
   end
 
+  scenario "see message in the correct order" do
+    login_with_oauth
+    visit root_path
+    create_scheduled_messages
+
+    visit scheduled_messages_path
+
+    expect(page).to have_selector("table > tr:nth-child(2) > td:nth-child(2)",
```

Comment:
sometimes when I have tricky expectations like this I like to wrap them in well-named methods that reveal their intentions, so for example you could do:

```ruby
expect(page).to have_message_in_order(message: first_scheduled_message, order: 1)
expect(page).to have_message_in_order(message: second_scheduled_message, order: 2)
```

```ruby
def have_message_in_order(message:, order: )
  order = order + 1
  have_selector("table > tr:nth-child(#{order}) > td:nth-child(2)",
    text: get_formatted_time(message).strip)
end
```
what do you think?